x-pack/plugins/enterprise_search/server/lib/connectors/add_connector.ts

```diff
@@ -84,11 +84,63 @@ export const addConnector = async (
     connectorsIndicesMapping[`${CONNECTORS_INDEX}-v${CONNECTORS_VERSION}`]?.mappings?._meta
       ?.pipeline;
 
+  const currentTimestamp = new Date(Date.now()).toISOString();
```

Comment:
nitpick(non-blocking): You can just do `new Date().toISOSTring()`, the `Date.now()` is redundant.